lib/rubocop/cop/offense.rb

```diff
@@ -72,18 +72,23 @@ def initialize(severity, location, message, cop_name,
       # @return [Boolean]
       #   whether this offense is automatically corrected.
       def corrected
-        @status == :corrected
+        @status == :corrected || @status == :corrected_with_todo
       end
       alias corrected? corrected
 
+      def corrected_with_todo
+        @status == :corrected_with_todo
+      end
+      alias corrected_with_todo? corrected_with_todo
```

Comment:
I don't get the purpose of this alias.

Reply:
We want to output [Todo] instead of [Corrected] in the rubocop formatter output.
Having a different corrected state allows us to do that.

Reply:
The method above this one, `corrected`, is also aliased but I've done some digging in the history and found that there used to be an instance variable called `@corrected` with an `attr_reader :corrected` and then we needed the alias `corrected?` to get the nicely named predicate. Now that we have a `@status` instance variable, we could just define `corrected?` and `corrected_with_todo?` methods and no aliases.

Reply:
I like that, however perhaps that's more suited to being a separate PR?

Reply:
OK, I'll merge this one, so you can do a follow-up PR.